edition-archive-frontend/pages/soviet-survivors/search/index.vue

```diff
@@ -2,36 +2,87 @@
   <SovietSurvivorsLayout :headline="$t('search.label')">
 
     <template #content>
-      <div class="card text-center mb-2">
-        <div class="card-body">
-          <h5 class="card-title">{{ $t("search.basic") }}</h5>
+      <h3>{{$t("search.label")}}</h3>
+      <!-- search form-->
+      <div  class="row">
+        <div class="col-12 col-lg-8">
           <BasicSearchForm :searchString="model.searchString" v-on:search="changeURL"/>
         </div>
       </div>
 
-      <SolrPaginator v-on:pageChanged="pageChangedCallback"
-                     :count="model.count"
-                     :start="model.start"
-                     :numPerPage="20"/>
+      <!-- result list and filter -->
+      <div class="row results" v-if="model.result?.response?.docs?.length>0">
 
-      <div v-if="model.result?.response?.docs?.length>0">
-        <div class="search-result card mt-2 mb-2" v-for="(doc, index) in model.result.response.docs" :key="doc.id">
+        <!-- results -->
+        <div class="col-12 col-lg-8 order-2 order-lg-1 results__list">
 
+          <!-- results: headline and sorting-->
+          <div class="row g-0 result-options">
+            <div class="col hit-count ms-3">
+              <h2>{{ $t('search.hitCount', { count: model.count }) }}</h2>
+            </div>
+          </div>
 
-          <div class="card-body">
-            <nuxt-link :to="`/soviet-survivors/documents/${getMyCoReIdNumber(doc['id'])}?search=${model.searchString}&start=${model.start + index}`" class="main-title">
-              {{ doc["mods.title.main"] }}
-            </nuxt-link>
-            <div v-if="doc['mods.abstract']?.length>0">{{ trimString(doc["mods.abstract"][0]) }}</div>
+          <div class="result_list">
+            <div id="hit_list">
+
+              <div
+                class="hit_item"
+                v-for="(doc, index) in model.result.response.docs" :key="doc.id">
+
+
+                <div class="row hit_item_head">
+                  <div class="col-12">
+                    <div class="hit_counter">{{index + 1 + model.start}}</div>
+                  </div>
+                </div>
+
+                <div class="row hit_item_body">
+                  <div class="col-12">
+                    <div class="hit_download_box">
+                      <a title="Sammelwerk"
```

Comment:
"Sammelwerk" ist definitiv falsch - hier könnte einfach "Vorschaubild" oder so etwas in der Art stehen?

Reply:
Auf [ Vorschau | Preview | Aperçu ] geändert.